src/platform_linux/laio.c

```diff
@@ -4,7 +4,14 @@
 /*
  * laio.c --
  *
- *     This file contains the inplementation for a libaio wrapper.
+ *     This file contains the implementation for a libaio wrapper.
+ *
+ * The external callable interfaces are defined in io.h. This module
+ * supports both synchronous and async IO.
+ *
+ * - Sync  IO interfaces: io_read(), io_write()
+ * - Async IO interfaces: io_read_async(), io_write_async()
+ * - Async IO completion interfaces: io_cleanup(), io_cleanup_all()
```

Comment:
I think this comment block doc'ing sync / async interfaces is reasonably complete.

If you'd like to see more doc'ed, kindly list the things that should be included.

Reply:
Added these comments to help better doc the API.